command/connect/envoy/envoy.go

```diff
@@ -47,9 +36,6 @@ type cmd struct {
 	http   *flags.HTTPFlags
 	help   string
 	client *api.Client
-	// DirectOut defaults to os.stdout but is a property to allow capture during
-	// tests to have more useful output.
-	directOut io.Writer
```

Comment:
If you don't mix up informational messages with output then the underlying OutputWriter construct is already doing this for us.